src/yadism/coefficient_functions/light/g4_nc.py

```diff
@@ -0,0 +1,51 @@
+from ..partonic_channel import RSL
+from . import nlo
+from . import partonic_channel as pc
+
+
+class NonSinglet(pc.LightBase):
+    @staticmethod
+    def LO():
+        """
+        |ref| APFEL
+        """
+
+        # leading order is just a delta function
+        return RSL.from_delta(1.0)
+
+    @staticmethod
+    def NLO():
+        """
+        |ref| hep-ph/9412255
+        """
+
+        return RSL.from_distr_coeffs(
+            nlo.g4.ns_reg, (nlo.g4.ns_delta, nlo.g4.ns_omx, nlo.g4.ns_logomx)
+        )
+
+    def NNLO(self):
+        return None
+
+
+class Gluon(pc.LightBase):
+    def NLO(self):
+        r"""
+        |ref| APFEL
+
+        Note
+        ----
+        2 * n_f is coming from momentum sum rule q_i -> {q_i, g} but
+        g -> {g, q_i, \bar{q_i} forall i}, so the 2 * n_f is needed
+        to compensate for all the number of flavours plus antiflavours
+        in which the gluon can go.
+        """
+
+        return RSL(nlo.g1.gluon_reg, args=[self.nf])
```

Comment:
I think that the `g1_gluon`-contribution is zero (same as `g1_gluon`). But I am not sure if returning zero in `nlo.g4.gluon_reg`  would be the same as calling `pc.EmptyPartonicChannel`. cc @felixhekhorn, @AleCandido 

```suggestion
class Gluon(pc.EmptyPartonicChannel):
    pass
```

Reply:
You can look at the definition: `EmptyPartonicChannel` is just a marker, but adds nothing to the base class:
https://github.com/NNPDF/yadism/blob/46ca667626d5f98f98d9e7a01566ec1a7f3d0834/src/yadism/coefficient_functions/partonic_channel.py#L171-L173

and the base class definitions for the orders is pretty simple:
https://github.com/NNPDF/yadism/blob/46ca667626d5f98f98d9e7a01566ec1a7f3d0834/src/yadism/coefficient_functions/partonic_channel.py#L154-L160